src/borg/helpers.py

```diff
@@ -391,15 +392,15 @@ def parse_timestamp(timestamp):
         return datetime.strptime(timestamp, '%Y-%m-%dT%H:%M:%S').replace(tzinfo=timezone.utc)
 
 
-def parse_add_pattern(patternstr, roots, patterns, fallback):
+def parse_add_pattern(patternstr, roots, ie_patterns, fallback):
     """Parse a pattern string and add it to roots or patterns depending on the pattern type."""
-    pattern = parse_inclexcl_pattern(patternstr, fallback=fallback)
-    if pattern.ptype is RootPath:
-        roots.append(pattern.pattern)
-    elif pattern.ptype is PatternStyle:
-        fallback = pattern.pattern
+    ie_pattern = parse_inclexcl_command(patternstr, fallback=fallback)
+    if ie_pattern.cmd is IECommand.RootPath:
```

Comment:
Either will work -- but enum values are singletons, so `is` works fine, and I believe it is the "right way" to compare them.